settings/dev.py

```diff
@@ -10,7 +10,19 @@
     'debug_toolbar.middleware.DebugToolbarMiddleware',
 ]
 
-ALLOWED_HOSTS = [os.environ.get('MY_HOST_IP'), 'api-staging-art.andela.com']
+
+ALLOWED_HOSTS = [os.environ.get('MY_HOST_IP'), 'api-staging-art.andela.com',
+                 '127.0.0.1']
+
+CORS_ORIGIN_WHITELIST = (
+    'art-dashboard-staging.herokuapp.com',
+    '127.0.0.1',
+    'staging-art.andela.com'
+)
```

Comment:
the regex takes care of '127.0.0.1' and 'staging-art.andela.com'

Reply:
So there is no need to have them in the CORS_ORIGIN_WHITELIST? @egichuri

Reply:
yeah.. we can have only 'art-dashboard-staging.herokuapp.com' there